.github/workflows/release.yaml

```diff
@@ -0,0 +1,122 @@
+          git push origin --tags
+        
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Quay
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: quay.io/gingersnap/operator:${{ env.release-version }}
+          build-args: |
+            OPERATOR_VERSION=${{ env.release-version }}
+
+      - name: Create OLM Bundle
+        run:  |
+            make operator-sdk
+            make bundle VERSION=${{ env.release-version }} CHANNELS=alpha DEFAULT_CHANNEL=alpha IMG=quay.io/gingersnap/operator:${{ env.release-version }}
+
+      - name: Checkout community-operators repo
+        uses: actions/checkout@v2
+        with:
+          repository: k8s-operatorhub/community-operators
+          path: community-operators
+          ref: main
+
+      - name: Copy contents of bundle/ 
+        run: |
+          mkdir -p community-operators/operators/gingersnap/${{ env.release-version }}
+          cp -R bundle/* community-operators/operators/gingersnap/${{ env.release-version }}
+          git add community-operators/operators/gingersnap/${{ env.release-version }}
+          git commit -m "Operator Bundle ${{ env.release-version }}"
+
+      - name: Create Pull Request to community-operators repo from forked repository 
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with: 
+          token: ${{ secrets.PAT_TOKEN }}
+          path: community-operators
+          commit-message: 'Gingersnap Operator ${{ env.release-version }}'
+          committer: infinispan-qe-bot <q*@infinispan.org>
+          author: infinispan-qe-bot <q*@infinispan.org>
+          branch: pr_request
+          delete-branch: true
+          push-to-fork: infinispan-qe-bot/community-operators
+          title: 'Gingersnap Operator ${{ env.release-version }}'
+
+      - name: Checkout community-operators-prod repo
+        uses: actions/checkout@v2
+        with:
+          repository: redhat-openshift-ecosystem/community-operators-prod
+          path: community-operators-prod
+          ref: main
+
+      - name: Copy contents of bundle/ and issue PRs
+        run: |
+          mkdir -p community-operators-prod/operators/gingersnap/${{ env.release-version }}
+          cp -R bundle/* community-operators-prod/operators/gingersnap/${{ env.release-version }}
+          git add community-operators-prod/operators/gingersnap/${{ env.release-version }}
+          git commit -m "Operator Bundle ${{ env.release-version }}"
```

Comment:
How about: "Gingersnap ${{ env.release-version }}"?

Reply:
yes . this commit message is better . updated